src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -617,6 +629,31 @@
     text: az sql midb show -g mygroup --mi myinstance -n mymanageddb
 """
 
+helps['sql midb retention-policy'] = """
+type: group
+short-summary: Manage SQL managed instance database backup retention policy.
+"""
+
+helps['sql midb retention-policy update'] = """
+type: command
+short-summary: Update short term retention for automated backups on a single database.
```

Comment:
Hi, I am trying to understand your service.  And I am a little confused about the retention policy here. Can you help me better understand your service? I am not sure if your service also has delete retention policy and restore policy. And for your retention policy days, is it defined to specify the days how long the deleted sql database will be retained or restored?

Reply:
We are taking automated backups for every database on MI. Automated backups can be used to restore databases to any point withing the retention period. This policy will determine for how long are we going to keep those backups before deleting them(retention period). 

There is only one policy per database, and it ranges for 7 to 35 for live databases, and 0 to 35 for dropped databases. Note we are going to reduce minimum retention days for live databases to 1 in the future.